content/20.results.md

```diff
@@ -98,3 +98,16 @@ Outside of the primary range of our analyses, the two names of 2020 PSB keynote
   For each region, the mean predicted probability of Pubmed articles is shown as teal LOESS curve, and the mean probability and 95% confidence interval of the ISCB honoree predictions are shown as dark circles and vertical lines (B).
 
 ](https://raw.githubusercontent.com/greenelab/iscb-diversity/master/figs/region_breakdown.png){#fig:region_breakdown}
+
+
+### Assessing Affiliation Diversity of Authors and Honorees
+We observed an increasing proportion of author affiliation located in East Asia & Pacific (Fig. {@fig:affiliation_breakdown}A, left).
+This was primarily driven by publications in Bioinformatics and BMC Bioinformatics (see [notebook](https://greenelab.github.io/iscb-diversity/15.analyze-affiliation.html#sup_fig_s6)).
+Among ISCB keynote speakers and fellows, we found that the majority of honorees are affiliated with institutions and companies from North America (Fig. {@fig:affiliation_breakdown}A, right).
+Noting the small sample size, we found discrepancies between the group of honoree composition versus PubMed. 
+Compared to Pubmed authors, ISCB keynote speakers and fellows have a higher proportion of individuals affiliated with North American institutions and companies (Fig. {@fig:affiliation_breakdown}B) and lower proportion of individuals affiliated with institutions or companies from Europe & Central Asia and East Asia & Pacific. 
+
+![Compared to Pubmed authors, honorees affiliated with institutions from North America are overrepresented while those from Europe and Central Asia, East Asia & Pacific group are underrepresented.
+Estimated composition of affiliation locations over the years of (A, left) all Pubmed computational biology and bioinformatics journal authors, and (A, right) all ISCB Fellows and keynote speakers was computed as the geographical proportions of Pubmed articles or ISCB honorees affiliations each year.
+For each region, the geographical proportions of Pubmed articles affiliations are shown as teal LOESS curves, and the geographical proportions and 95% confidence interval of the ISCB affiliations are shown as dark circles and vertical lines (B).
+](https://raw.githubusercontent.com/greenelab/iscb-diversity/master/figs/region_breakdown.png){#fig:affiliation_breakdown}
```

Comment:
What do the letters correspond to in this figure? The country groupings in

- [`2020-01-31_natural-earth-world-map.png`](https://raw.githubusercontent.com/greenelab/iscb-diversity/b41b8fad157d0b878a2476cec998aba0643742bf/figs/2020-01-31_natural-earth-world-map.png)
- the nameprism groupings from Figure 1?

If the former, it's a bit confusing to have letters refer to different groupings. Could we just replace with the standard world geographical mapping region labels?

Reply:
My bad. The linked figure is incorrect. It should be fixed now.